docs/management/admin/response-actions.asciidoc

```diff
@@ -92,6 +92,18 @@ Required privilege: *Process Operations*
 
 Example: `suspend-process --pid 123 --comment "Suspend suspicious process"`
 
+=== `get-file`
+
+Retrieve a file from a host. You must include the following parameter to specify the file's location:
+
+* `--path` : The file's full path (including file name).
+
+Required privilege: *File Operations*
+
+Example: `get-file --path "/full/path/to/file.txt" --comment "Possible malware"`
+
+NOTE: When {elastic-defend} prevents file activity due to <<malware-protection,malware prevention>>, the file is quarantined on the host and a Malware Prevention Alert is created. To retrieve this file with `get-file`, copy the path from the alert's *Quarantined file path* field (`file.Ext.quarantine_path`) and paste it into the `--path` parameter.
```

Comment:
Wondering if `Malware Prevention Alert` needs to be title case? I know it's in the UI in the alert details panel, but in a general reference, I'm thinking it should be lowercase.